pkgs/development/libraries/wcslib/default.nix

```diff
@@ -0,0 +1,18 @@
+{ fetchurl,  stdenv, flex }:
+
+
+stdenv.mkDerivation rec {
+  name = "wcslib-4.25";
+
+  buildInputs = [ flex ];
+
+  src = fetchurl {
+    url = "ftp://ftp.eso.org/pub/dfs/pipelines/libraries/wcslib/wcslib-4.25.tar.bz2";
```

Comment:
If you're using `rec` above (currently for no reason), we might as wel use ${name} here.